src/main/java/org/motechproject/newebodac/mapper/UserMapper.java

```diff
@@ -0,0 +1,22 @@
+package org.motechproject.newebodac.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.ReportingPolicy;
+import org.mapstruct.factory.Mappers;
+import org.motechproject.newebodac.domain.security.User;
+import org.motechproject.newebodac.dto.UserDto;
+
+@Mapper(uses = { UuidMapper.class },
+    unmappedTargetPolicy = ReportingPolicy.IGNORE)
+public interface UserMapper extends EntityMapper<UserDto, User> {
+
+  UserMapper INSTANCE = Mappers.getMapper(UserMapper.class);
+
+  @Mapping(target = "password", ignore = true)
+  @Mapping(target = "id", ignore = true)
+  @Mapping(target = "createDate", ignore = true)
+  @Mapping(target = "updateDate", ignore = true)
+  void update(UserDto userDto, @MappingTarget User user);
+}
```

Comment:
you will need to add mappers for UserRole, check how it's done in RoleMapper for Permissions